tests/Command/CopyConnectionSchemaCommandTest.php

```diff
@@ -0,0 +1,144 @@
+<?php
+declare(strict_types=1);
+
+namespace Kununu\TestingBundle\Tests\Command;
+
+use Doctrine\DBAL\Connection;
+use Kununu\TestingBundle\Command\CopyConnectionSchemaCommand;
+use Kununu\TestingBundle\Service\SchemaCopy\SchemaCopyAdapterFactoryInterface;
+
+/**
+ * @group legacy
+ */
+final class CopyConnectionSchemaCommandTest extends AbstractCommandTestCase
+{
+    private const COMMAND = 'kununu_testing:connections:schema:copy';
+
+    private const TABLES = [
+        'doctrine_migration_versions',
+        'table_1',
+        'table_2',
+        'table_3',
+        'table_to_exclude',
+    ];
+
+    private const VIEWS = [
+        'my_view',
+    ];
+
+    private $adapter;
+
+    public function testCommandValidNonInteractive(): void
+    {
+        $this->executeCommand('monolithic', 'monolithic_test', '', false);
+
+        $this->assertEquals(0, $this->commandTester->getStatusCode());
+        $this->assertEquals(self::TABLES, $this->adapter->getTables());
+        $this->assertEquals(self::VIEWS, $this->adapter->getViews());
+    }
+
+    public function testCommandValidInteractiveConfirmed(): void
+    {
+        $this->executeCommand('monolithic', 'monolithic_test');
```

Comment:
I think having the `$confirmation` "yes" would improve a bit the readability of the test

Reply:
It's the default value : `private function executeCommand(?string $from, ?string $to, string $confirmation = 'yes', bool $interactive = true): void`